src/Drupal/V8/BasicCommands.php

```diff
@@ -144,6 +144,7 @@ public function make($opts = [
         $composerTask->run();
 
         // Symlink dirs from src into docroot.
+        $this->docrootSymlink('src/config', 'docroot/config');
```

Comment:
Doesn't our default config place the config directory at the root of the repo, not in docroot?